framework/Data/SqlMap/Statements/TMappedStatement.php

```diff
@@ -512,7 +512,7 @@ protected function applyResultMap($row, &$resultObject = null)
 		} else {
 			$obj = $this->fillDefaultResultMap(null, $row, $resultObject);
 		}
-		if (class_exists('TActiveRecord', false) && $obj instanceof TActiveRecord) {
+		if (class_exists('\Prado\Data\ActiveRecord\TActiveRecord', false) && $obj instanceof TActiveRecord) {
```

Comment:
Precisely, in this case, you can't use a `PradoDataActiveRecordTActiveRecord::class` which would load the class, whereas the false is precisely there to ask you not to do it in case it doesn't exist ;)